include/broadcom/bcm_board_types.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _BCM_BOARD_TYPES_H_
```

Comment:
Should you add copyright information here?

Reply:
Yeah. right...